eks-distro-base/update_base_image.sh

```diff
@@ -24,8 +24,12 @@ IMAGE_NAME=$2
 AL_TAG=$3
```

Comment:
NAB: rename this to `TAG` instead?

Reply:
let me think about that, its a good call but the AL_TAG concept is everywhere so id like to change it completely if going to do that.